notebooks/run_model_on_snellius.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    for ncfile in Path(path_dict["ForcingPath"]).iterdir():
</pre><p>A suggestion to refactor this block: </p><p><code>forcing_filenames_list = ["NL-Hor_2008-2011_FLUXNET2015_Met.nc", "ZA-Kru_2000-2002_FLUXNET2015_Met.nc"]</code></p><p> </p><p><code>if not forcing_filenames_list:</code></p><p><code>    forcing_filenames_list = Path(path_dict["ForcingPath"]).iterdir():</code></p><p> </p><p><code>for ncfile in forcing_filenames_list:</code></p><p><code>    station_name = ncfile.stem.split('_')[0]</code></p><p><code>    input_dir(station_name)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/processing/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>